content/kubermatic/master/installation/install_kkp_on_node/_index.en.md

```diff
@@ -0,0 +1,96 @@
++++
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone"
+date = 2022-02-07T12:07:15+02:00
+weight = 10
+enableToc = true
++++
+
+# Get started with KKP - “KKP on single node with Kubeone”
+
+This chapther will guide you through the KKP Master setup on a single master/worker k8s node using Kubeone. Here, we are leveraging the kubeone addons capability for quick installation of KKP.
```

Comment:
IMO chapther should be chapter.

Reply:
Thanks @stroebitzer. Fixed the typo error!